.travis.yml

```diff
@@ -30,7 +30,9 @@ virtualenv:
 install:
   - pip install -q unidecode
   - pip install unicodecsv
+  - pip install xlrd
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone https://github.com/OCA/bank-statement-import.git ${HOME}/build/CompassionCH/bank-statement-import -b 8.0
```

Comment:
To add bank-statement-import, please add it in oca_dependencies.txt

https://github.com/OCA/l10n-switzerland/blob/7.0/oca_dependencies.txt